economicsl/__init__.py

```diff
@@ -23,7 +30,7 @@ def __init__(self):
         self.mailbox = Mailbox(self)
 
     def send_obligation(self, recipient, obligation: Obligation) -> None:
-        recipient.receive_message(obligation)
+        self.postbox.append((recipient, obligation))
         self.mailbox.add_to_obligation_outbox(obligation)
```

Comment:
is self mailbox, some kind of outbox?

Reply:
`self.mailbox` contains `self.mailbox.obligation_unopened`, `self.mailbox.obligation_outbox`, `self.mailbox.obligation_inbox`. There used to be goods and cash inbox, but they are removed since they are unused.